packages/types/src/scope.ts

```diff
@@ -89,6 +89,16 @@ export interface Scope {
    */
   setSpan(span?: Span): this;
 
+  /**
+   * Returns the `Span` if there is one
+   */
+  getSpan(): Span | undefined;
```

Comment:
Okay, only now I realized that `getSpan` already exists on the scope, but is documented as `@hidden`.

I believe we should either not include this or mark it as `@hidden` as well or make `Scope.getSpan` completely public and documented.